From d087c4186e64bdea8c10a2ab3fad9f2474e9c818 Mon Sep 17 00:00:00 2001
From: Tom Rondeau <tom@trondeau.com>
Date: Fri, 11 Apr 2014 11:47:58 -0400
Subject: [PATCH 16/17] qtgui: fixed frequency scale for float frequency
 plots.

Was setting bandwidth to half what it should have. Tested QWT 5.2.0 - 6.1.0.
---
 gr-qtgui/lib/freq_sink_f_impl.cc |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gr-qtgui/lib/freq_sink_f_impl.cc b/gr-qtgui/lib/freq_sink_f_impl.cc
index b77e523..b2054bc 100644
--- a/gr-qtgui/lib/freq_sink_f_impl.cc
+++ b/gr-qtgui/lib/freq_sink_f_impl.cc
@@ -148,7 +148,7 @@ namespace gr {
       d_main_gui = new FreqDisplayForm(d_nconnections, d_parent);
       set_fft_window(d_wintype);
       set_fft_size(d_fftsize);
-      set_frequency_range(d_center_freq, d_bandwidth/2.0);
+      set_frequency_range(d_center_freq, d_bandwidth);
 
       // initialize update time to 10 times a second
       set_update_time(0.1);
-- 
1.7.10.4

